client/landing/gutenboarding/stores/onboard/types.ts

```diff
@@ -34,4 +34,5 @@ export interface Design {
 	template: string;
 	fonts: FontPair;
 	categories: Array< string >;
+	premium: boolean;
```

Comment:
Even if this API is currently used just in our static list, since it's a boolean we should name it `is_premium` 🤔